### PR TITLE
Stop calling the workstation toolbar experimental

### PR DESCRIPTION
**Pull request by tas50**
It's time to remove the experimental word here

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: omnibus/resources/chef-workstation/pkg/welcome.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@ Including:
 <ul>
   <li><strong>chef-run:</strong> Ad-hoc remote execution of Chef</li>
   <li><strong>ChefDK:</strong> Tools for local Chef Development</li>
```
Comment (afiune) [inline omnibus/resources/chef-workstation/pkg/welcome.html.erb]: Though, we don't ship ChefDK anymore, do we?

Comment (tas50) [reply omnibus/resources/chef-workstation/pkg/welcome.html.erb]: I didn't want to touch the issue of DK being bundled in this PR. That's written all over the place.

Comment (afiune) [reply omnibus/resources/chef-workstation/pkg/welcome.html.erb]: Agreed, just wanted to bring it up 👍

**Comment by tyler-ball** [comment]
Connected to https://github.com/chef/chef-workstation/issues/536

**Comment by marcparadise** [comment]
Looks good to me, I've requested @chef/docs-team to take a look too.

**Comment by tas50** [comment]
😄

**Pull request merged**